extensions/plugins/system/redcore/redcore.php

```diff
@@ -90,6 +90,11 @@ public function onAfterInitialise()
 						$data = RApi::getPostedData();
 						$dataGet = $input->get->getArray();
 
+						if (!empty($data['language']) && empty($dataGet['language']))
```

Comment:
It wasn't working for SOAP, because we're not actually sending anything that JInput can interpret in the $_POST vars.
  
We could add a check to see if the option is turned on, but we do need to manipulate the $db->translation value after we get the xml data out and can confirm there is a language param. Maybe we could move this to the SOAP API constructor.